### PR TITLE
Add SXP 10.4 connector modules

### PR DESCRIPTION
**Pull request by sc-marcusheath**
Add latest connector module releases, compatible with SXP 10.4.

### CODE REVIEW

File: compose/def/9.0/ltsc2022/xm1/docker-compose.override.yml
Comment (sc-jennengng) [inline compose/def/9.0/ltsc2022/xm1/docker-compose.override.yml:1]: Hi Marcus, can you help to remove the def 9.0 xm1 script? We no longer support DEF on xm1

File: k8s/def/9.0/ltsc2022/overrides/xm1/kustomization.yaml
Comment (sc-jennengng) [inline k8s/def/9.0/ltsc2022/overrides/xm1/kustomization.yaml:1]: Can you help to remove this def 9.0 xm1 script too. Thanks

**Review by sc-jennengng — APPROVED** [review]

File: compose/def/9.0/ltsc2022/xm1/docker-compose.override.yml
Comment (sc-marcusheath) [reply compose/def/9.0/ltsc2022/xm1/docker-compose.override.yml:1]: Removed two files plus their parent xm1 folder. Is this ok?

File: k8s/def/9.0/ltsc2022/overrides/xm1/kustomization.yaml
Comment (sc-marcusheath) [reply k8s/def/9.0/ltsc2022/overrides/xm1/kustomization.yaml:1]: Removed the kustomization script.

File: compose/def/9.0/ltsc2022/xm1/docker-compose.override.yml
Comment (sc-jennengng) [reply compose/def/9.0/ltsc2022/xm1/docker-compose.override.yml:1]: yes, thanks

**Review by sc-jennengng — APPROVED** [review]

**Pull request merged**